javascript/code-challenges/challenges-01.test.js

```diff
@@ -0,0 +1,219 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1
+
+Write a function named `addOne` that takes an array of numbers, and returns a new array of the numbers, incremented by 1.
+
+Use `forEach` to loop over the input array and work with each value.  Push the new value into a local array. Return the local array;
+------------------------------------------------------------------------------------------------ */
+
+const addOne = (arr) => {
+  let newArray= [];
+  arr.forEach(value =>{
+    value++;
+    newArray.push(value);
+  });    return newArray;
+
+};
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 2
+
+Write a function named `addExclamation` that takes an array of strings, and returns a new array of the same strings with an "!" added to the end.
+
+Use `forEach` to loop over the input array. Modify each string, and add the updated value into a local array. Return the local array;
+------------------------------------------------------------------------------------------------ */
+
+const addExclamation = (arr) => {
+  let localArray = [];
+  arr.forEach(string =>{
+    let newString =`${string}!`;
+    localArray.push(newString);
+   
+  });
+   return localArray;
```

Comment:
Make sure you have consistent indentation. Your indentation is all over the place.